### PR TITLE
Updated for 2017

### PR DESCRIPTION
**Pull request by amboyscout**
Fixed version numbers for requirements and the download location for WPILib Eclipse Plugins.

### CODE REVIEW

File: robotpy.feature/feature.xml [hunk 1]
```diff
@@ -104,7 +104,7 @@ This Agreement is governed by the laws of the State of New York and the intellec
    </license>
 
    <requires>
-      <import feature="org.python.pydev.feature" version="3.9.1" match="greaterOrEqual"/>
+      <import feature="org.python.pydev.feature" version="6.0.0.201709191431" match="greaterOrEqual"/>
```
Comment (Twinters007) [inline robotpy.feature/feature.xml]: This could probably be greater or equal to 6.0.0

File: robotpy.feature/feature.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <feature
       id="io.github.robotpy.plugins.robotpy.feature"
       label="Robot Python Development"
-      version="2015.0.8"
+      version="2017.1.0"
```
Comment (Twinters007) [inline robotpy.feature/feature.xml]: Why not 1.2? That's the latest release

File: robotpy/resources/py/preexec.py [hunk 3]
```diff
@@ -16,9 +16,9 @@
 is_windows = hasattr(sys, 'getwindowsversion')
 
 # setup minimum requirements here 
-PYFRC_MIN_VERSION = LooseVersion("2015.2.0")
-WPILIB_MIN_VERSION = LooseVersion("2015.0.6")
-NETWORKTABLES_MIN_VERSION = LooseVersion("2015.0.6")
+PYFRC_MIN_VERSION = LooseVersion("2017.1.5")
+WPILIB_MIN_VERSION = LooseVersion("2017.3.1")
```
Comment (Twinters007) [inline robotpy/resources/py/preexec.py]: There is no 3.1 is there?

Comment (amboyscout) [reply robotpy/resources/py/preexec.py]: There is a 3.1 It came out in may afaik

File: robotpy.feature/feature.xml (hunk 1 above)
Comment (amboyscout) [reply robotpy.feature/feature.xml]: Ill change that

File: robotpy.feature/feature.xml (hunk 2 above)
Comment (amboyscout) [reply robotpy.feature/feature.xml]: Fair enough, will change that right now

File: robotpy/resources/py/preexec.py (hunk 3 above)
Comment (Twinters007) [reply robotpy/resources/py/preexec.py]: Is this....min robotpy version or wpilibj version?

**Comment by virtuald** [comment]
Now I'll have to remember how to push a new release...

**Comment by virtuald** [comment]
@amboyscout looks like the plugins don't actually compile? Can you take a look: https://travis-ci.org/robotpy/robotpy-eclipse-plugins/builds/288369103 

Thanks!

**Comment by Twinters007** [comment]
https://github.com/fabioz/Pydev/blob/master/plugins/org.python.pydev.core/src/org/python/pydev/core/IPythonNature.java#L43

Looks like pydev has been updated.

**Pull request merged**